climpred/utils.py

```diff
@@ -97,3 +103,52 @@ def reduce_time_series(forecast, reference, nlags):
     forecast = forecast.where(forecast.time >= imin, drop=True)
     reference = reference.where(reference.time >= imin, drop=True)
     return forecast, reference
+
+
+def assign_climpred_compute_to_attrs(
+    skill,
+    ds,
+    function_name='compute_perfect_model',
```

Comment:
Could this be defaulted to `None`? I don't see logically why the default should be `compute_perfect_model`.

Reply:
Or maybe not even a keyword arg, just a required position arg?